packages/venia-concept/src/components/Navigation/navigation.js

```diff
@@ -1,327 +1,143 @@
-import React, { PureComponent } from 'react';
-import { bool, func, object, shape, string } from 'prop-types';
+import React, { useCallback, useEffect, useState } from 'react';
+import { bool, func, number, objectOf, shape, string } from 'prop-types';
 
-import classify from 'src/classify';
-import Button from 'src/components/Button';
-import CreateAccount from 'src/components/CreateAccount';
-import SignIn from 'src/components/SignIn';
-import ForgotPassword from 'src/components/ForgotPassword';
-import CategoryTree from './categoryTree';
+import { mergeClasses } from 'src/classify';
+import AuthBar from 'src/components/AuthBar';
+import AuthModal from 'src/components/AuthModal';
+import Tree from './categoryTree';
 import NavHeader from './navHeader';
 import defaultClasses from './navigation.css';
-import { MyAccountMenuTrigger } from '../MyAccountMenuPage';
```

Comment:
How do you feel about cleaning up/removing all the stuff that isn't used anymore? Like this component and possibly others I didn't find yet :D